lib/git_reflow.rb

```diff
@@ -99,8 +99,12 @@ def deliver(options = {})
 
           if committed
             say "Merge complete!", :success
-            deploy_and_cleanup = ask "Would you like to push this branch to your remote repo and cleanup your feature branch? "
-            if deploy_and_cleanup =~ /^y/i
+
+            # check if user always wants to push and cleanup, otherwise ask
+            always_deploy_and_cleanup = GitReflow::Config.get('reflow.always-deploy-and-cleanup') == "true"
```

Comment:
How does the user set this?  Manually?  If so, that's fine, we'll just need to add supporting documentation for it